gn3/llms/process.py

```diff
@@ -43,32 +77,23 @@ def parse_context(context, get_info_func, format_bib_func):
         doc_info = get_info_func(doc_ids)
```

Comment:
Follow up to https://github.com/genenetwork/genenetwork3/pull/188/files#r1740678411
```
try:
  doc_info = get_info_func(doc_ids)
  bib_info = format_bib_func(doc_info)
except KeyError:
  bib_info = doc_ids
```

Reply:
Commit fixes this: https://github.com/genenetwork/genenetwork3/pull/188/commits/c16c54759cfd493250424ee3f565862e5d6009b3